### ISSUE TITLE
CI/CD: test-schedule-runner Exception raised: Guest couldn't be provisioned: Artemis resource ended in 'error' state

### ISSUE THREAD

**Issue by dougsland** [containers/qm#458]
Test farm is unable to run CI/CD jobs.

**Comment by dougsland** [comment]
@Yarboa are you familiar with this error ? Or are you working on this already? I remember you mentioning that you are working an issue in CI/CD.

**Comment by Yarboa** [comment]
Yes it seems that previous fix in testing-farm artemis 
with the fix, reverted from testing-farm from version

from testing-farm team 
```
this looks like incorrect version of Artemis is deployed at http://artemis.production.testing-farm.io/. the bugfix was released as an extra release, v0.0.70.2, but the package version reported by /about is 0.0.70
``` 

They have to find the reason for the rollback

**Comment by Yarboa** [comment]
Closed, this happened due to non latest artemis version in testing-Farm
